experimental/addons/mongodb/template.cue

```diff
@@ -0,0 +1,60 @@
+package main
+
+
+output: {
+        apiVersion: "core.oam.dev/v1beta1"
+        kind:       "Application"
+        spec: {
+                components: [
+                        {
+                                type: "k8s-objects"
+                                name: "mongodb-ns"
+                                properties: objects: [{
+                                        apiVersion: "v1"
+                                        kind:       "Namespace"
+                                        metadata: name: parameter.namespace
+                                }]
+                        },
+                        {
+                            name: "mongodb-helm"
+                            type: "helm"
+                            dependsOn: ["mongodb-ns"]
+                            type: "helm"
+                            properties: {
+                                    repoType: "helm"
+                                    url:      "https://charts.bitnami.com/bitnami"
+                                    chart:    "mongodb"
+                                    targetNamespace: parameter["namespace"]
+                                    version:  "13.1.2"
+                                    values: {
+                                            persistence: {
+                                                enabled: false
```

Comment:
Why disable the persistence?

Reply:
As an experimental addon, it may be easier and no dependencies for user to go.

Reply:
Can we let user determine whether disable the persistence?